controllers/cloud.redhat.com/providers/autoscaler/router.go

```diff
@@ -0,0 +1,33 @@
+package autoscaler
+
+import (
+	crd "github.com/RedHatInsights/clowder/apis/cloud.redhat.com/v1alpha1"
+	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/config"
+	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers"
+)
+
+//autoScaleProviderRouter is a wrapper for the different autoscaler providers.
+type autoScaleProviderRouter struct {
+	providers.Provider
+	Config config.DatabaseConfig
```

Comment:
Database Config?

Reply:
More copy/paste. Messy, messy. I wasn't sure what this was or if something else depended on it. Should have looked instead of just leaving it. Removed in next push.